### PR TITLE
[Magic WAN] Added IPsec to Aruba docs

### PR DESCRIPTION
**Pull request by dcpena**
Added IPsec sections to Aruba Partner docs to address [PCX-3927](https://jira.cfops.it/browse/PCX-3927).

### CODE REVIEW

File: content/magic-wan/partners/aruba-edgeconnect.md [hunk 1]
```diff
@@ -25,7 +25,13 @@ Before setting up a connection between EdgeConnect and Cloudflare, you must have
...
 - The EdgeConnect devices used in this tutorial and on v9.0. 
 
-## Example scenario
+## Example scenario 
+
+<details>
+<summary>
+  GRE tunnel configuration
+</summary>
+ <div class="special-class" markdown="1">
```
Comment (marciocloudflare) [inline content/magic-wan/partners/aruba-edgeconnect.md:34]: What's this `special-class` we have? :)

File: content/magic-wan/partners/aruba-edgeconnect.md [hunk 2]
```diff
@@ -69,9 +111,41 @@ To create the overlay policy:
...
+
+1. Create a compound application, which is a combination of all [Cloudflare public IPs](https://www.cloudflare.com/ips/) and ICMP packets.
+
+![Application definition screen with IP values](/magic-wan/static/app-definition.png)
+
+2. Create a breakout Business Intent Overlay (BIO) to bypass the IPsec tunnel as the first policy and use this newly created application as the match criteria. 
+
+3. Define at least one additional overlay policy and the traffic you want to send to Cloudflare over the IPsec tunnels.
+
+The service name used to send traffic through the tunnel created in the next step is **Cloudflare_IPsec**. The example uses **Match Everything** to send all other traffic through the established tunnel (both private east-west traffic & Internet bound north-south traffic through Cloudflare’s Secure Web Gateway).
```
Comment (marciocloudflare) [inline content/magic-wan/partners/aruba-edgeconnect.md]:
```suggestion
The service name used to send traffic through the tunnel created in the next step is **Cloudflare_IPsec**. The example uses **Match Everything** to send all other traffic through the established tunnel (both private east-west traffic and Internet bound north-south traffic through Cloudflare’s Secure Web Gateway). 
```

Comment (marciocloudflare) [reply content/magic-wan/partners/aruba-edgeconnect.md]: We don't have this in our style guide, but I figured that if we shouldn't use abbreviations, maybe we should also steer off from symbols? Wdyt?

File: content/magic-wan/partners/aruba-edgeconnect.md [hunk 3]
```diff
@@ -126,9 +260,45 @@ The traceroute shows the path going from the client (10.3.0.3) <br>
...
+</summary>
+ <div class="special-class" markdown="1">
+
+**Validate Secure Web Gateway**
+
+To validate traffic flow from the local subnet through Cloudflare’s Secure Web Gateway, perform a cURL as shown in the example below.
+
+![cURL example for validating traffic](/magic-wan/static/static-routes-west-ipsec.png)
+
+You can validate the request went through Secure Web Gateway with the presence of the `Cf-Team` response header or by looking at the logs in the dashboard under **Logs** > **Gateway** > **HTTP**.
```
Comment (marciocloudflare) [inline content/magic-wan/partners/aruba-edgeconnect.md]: Did you mean `You can validate *that* the request went through Secure Web Gateway` or `You can validate the request *sent* through Secure Web Gateway`?

**Review by marciocloudflare — APPROVED** [review]

File: content/magic-wan/partners/aruba-edgeconnect.md (hunk 1 above)
Comment (dcpena) [reply content/magic-wan/partners/aruba-edgeconnect.md:34]: It allows you to use markdown in the dropdowns which is handy for the images!

File: content/magic-wan/partners/aruba-edgeconnect.md (hunk 2 above)
Comment (dcpena) [reply content/magic-wan/partners/aruba-edgeconnect.md]: Good catch, thank you!

File: content/magic-wan/partners/aruba-edgeconnect.md (hunk 3 above)
Comment (dcpena) [reply content/magic-wan/partners/aruba-edgeconnect.md]: Fixed! thank you

**Pull request merged**